install/update_95_xx.php

```diff
@@ -47,6 +47,15 @@ function update95toXX() {
 
    include __DIR__ . '/update_95_xx/domains.php';
 
+   $to_add_uuid = ['Monitor', 'NetworkEquipment', 'Peripheral', 'Phone', 'Printer'];
```

Comment:
Only phone has been changed in empty sql file; should have been added for all types